src/printer.ts

```diff
@@ -101,9 +101,9 @@ export class PugPrinter {
 	private indentLevel: number = 0;
 
 	/**
-	 * The line length starts by 1, it's not an zero based index
+	 * Counting the line length starts at 0
 	 */
-	private currentLineLength = 1;
+	private currentLineLength = 0;
```

Comment:
<img src="https://user-images.githubusercontent.com/7195563/94650240-97fbbd00-02f6-11eb-9273-8cddb6be5915.png" width="150px" />

It's indeed index 1 :warning:  
Please try to start with an `currentLine*` based on `1`

Reply:
If you turn on the word wrap column in VSCode and set it to 80 and look at the tests that I added, you will see that you get the expected results with 0, not 1…

![image](https://user-images.githubusercontent.com/213275/94650659-6df6ca80-02f7-11eb-80b6-2b01582458f0.png)

Reply:
Ok need to check why I had used this index 1 base
Cause there also was another reason, but I don't know anymore

Need to start working _Home Office_, so I will only check time to time here :wave: 
Thanks for all our work :slightly_smiling_face:

Reply:
Of course, no worries! And thank you for the great work on this plugin, it's exactly what we need!

Reply:
Maybe you were using the `as-needed` option where the counting is off currently?

Reply:
Could be, need to look at the code again
It's a long time ago I read this part of code ^^'

Reply:
I know the feeling 😆